### ISSUE TITLE
Missing namespace when using AGP 8

### ISSUE THREAD

**Issue by rivella50** [PurpleSoftSrl/sumup_flutter_plugin#58]
When trying to build my app with AGP version 8.x and targetSdkVersion 34 i get this error:
```
A problem occurred configuring project ':sumup'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file: /Users/valley/.pub-cache/hosted/pub.dev/sumup-0.9.0/android/build.gradle. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.
```

What's missing is a namespace entry in android/build.gradle like this:
```
android {
    namespace "io.purplesoft.sumup"
    ...
}
```
Could you please add this missing part?
Thanks a lot.

**Comment by sstasi95** [comment]
Fixed in version 0.10.0

**Comment by rivella50** [comment]
Thanks a lot!
